fusor-ember-cli/app/routes/deployment.js

```diff
@@ -150,6 +150,11 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
       console.log(reason);
     },
 
+    refreshModel: function(){
+      console.log('refreshModelOnDeploymentRoute');
+      return this.refresh();
```

Comment:
this didn't work, but I'll leave it here anyway.
`this.refresh()` is supposed to refresh also the child routes.